source/PreferencesPanel.cpp

```diff
@@ -803,6 +807,11 @@ void PreferencesPanel::DrawSettings()
 			isOn = Preferences::GetAlertIndicator() != Preferences::AlertIndicator::NONE;
 			text = Preferences::AlertSetting();
 		}
+		else if(setting == COLOR_FILTER_SETTING)
+		{
+			text = Preferences::ColorFilterSetting();
+			isOn = text != "off";
```

Comment:
The text can never be "off", so just have isOn always be true.
```suggestion
			isOn = true;
			text = Preferences::ColorFilterSetting();
```